src/node/allPlayers.js

```diff
@@ -0,0 +1,39 @@
+const constants = require('./constants.js');
+const Player = require('./player.js');
+
+function AllPlayers() {
```

Comment:
Hmmm this abstraction is kinda weird, shouldn't the game hold the winner and control which player is next?
You wanted just to isolate the rules/functions that operates under players, right? These functions operating on player could just live inside player.js as plain functions, no need for this "AllPlayers" function grouper